app/main.py

```diff
@@ -1,34 +1,137 @@
+from typing import Any
+
+
+class ShipCountError(Exception):
+    pass
+
+
 class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
+    def __init__(
+        self,
+        row: int,
+        column: int,
+        is_alive: bool = False,
```

Comment:
Not sure about this, in the assignment was given initially is_alive: bool = True, which makes sense since when we have just created a Deck it is alive